lib/core/data_providers/cards_data_provider.dart

```diff
@@ -24,6 +24,9 @@ class CardsDataProvider extends ChangeNotifier {
       'availability',
       'events',
       'news',
+//      'special_events',
```

Comment:
commented out on purpose? If you've commented anything out as a part of dev/testing please revert them. If you're commenting something to save it for later. I'd recommend adding a todo so someone will remember to followup on it.

Reply:
Yup, thanks for catching those ! Have a bad practice of 'commenting out for later'. I have removed the extra comments in the most recent commit.

Reply:
The default cards API needed an update for the Parking card to activate. This has been completed in QA.

Re-testing on i8wu's branch, the Parking card is loading, but not the inner card content.

Err:
`Uncaught TypeError: Cannot read property 'split' of null", source: https://mobile.ucsd.edu/replatform/v1/qa/webview/parking/index.js (62)`

Reply:
@c3bryant Looks like the "spots" are all deactivated by default. If you press manage spots and toggle them all on content shows up. I'm seeing an undefined text on it though 
![image](https://user-images.githubusercontent.com/20545218/91980342-f4f65a00-ecdb-11ea-8d33-91f899bf3e04.png)

Reply:
I have fixed the 2 problems discussed above. If lotContext is not provided then we can display a nbsp. And added a default message when no spot types are selected by user or provided in the url.

![image](https://user-images.githubusercontent.com/29939574/91988289-b8af1200-ed4c-11ea-93fd-f1d73e347cb6.png)